deepface/detectors/Yolov8faceWrapper.py

```diff
@@ -0,0 +1,71 @@
+from deepface.detectors import FaceDetector
+
+# Models names and paths
+PATHS = {
+    "yolov8n": "/.deepface/weights/yolov8n-face.pt",
+}
+
+# Google Drive base URL
+BASE_URL = "https://drive.google.com/uc?id="
+
+# Models' Google Drive IDs
+IDS = {
+    "yolov8n": "1qcr9DbgsX3ryrz2uU8w4Xm3cOrRywXqb",
+}
+
+
+def build_model(model: str):
+    """Function factory for YOLO models"""
+    from deepface.commons.functions import get_deepface_home
+
+    # Get model's weights path and Google Drive URL
+    func_weights_path = f"{get_deepface_home()}{PATHS[model]}"
+    func_url = f"{BASE_URL}{IDS[model]}"
+
+    # Define function to build the model
+    def _build_model(weights_path: str = func_weights_path, url: str = func_url):
+        import gdown
+        import os
+
+        # Import the Ultralytics YOLO model
+        from ultralytics import YOLO
```

Comment:
this import requires to download a ultralytics python package. this should be mentioned in requirements.txt or requirements_additional.txt. if this is not a must dependency, then add it to requirements_additional.

Reply:
Probably not a must (like mediapipe). I have to add a meaningful import error.